src/ankita/assignment/assignment6/FillForm4.java

```diff
@@ -0,0 +1,81 @@
+/*
+Assignment - 6 : 13th Jan'2020
+Program 1 : a) fillup the form "First_Form_4" by taking data from properties file
+b) click on "Go and Practice", validate user should navigate to AutomationBykrishna.com
+c) go back in the history, verify firstname field contains the same value which is provided from properties file.
+*/
+package ankita.assignment.assignment6;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import ankita.base.PredefinedActions;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class FillForm4 {
+
+	public static void main(String[] args) throws IOException {
+		WebDriver driver = PredefinedActions
+				.start("H:\\JAVA selenium\\Selenium\\SeleniumTechnoOct2020\\resources\\forms\\First_Form_4.html");
+		File file = new File("./src/ankita/base/confData.properties");
+		FileInputStream inputStream = new FileInputStream(file);
+
+		Properties prop = new Properties();
+		prop.load(inputStream);
```

Comment:
Hi @AnkitaDOct2020 
For reading properties file code should written as separate method,